adscompstat/match.py

```diff
@@ -14,8 +14,22 @@
 
 
 class CrossrefMatcher(object):
-    def __init__(self):
-        pass
+    def __init__(self, related_bibstems=conf.get("RELATED_BIBSTEMS", [[]])):
+        self.related_bibstems = related_bibstems
+
+    def _compare_bibstems(self, testBibstem, classicBibstem):
```

Comment:
The comparisons are done using the substring `bibcode[4:9]`, so these will include the trailing dots for bibstems less than five characters long.  Ultimately I would like to serve these data from the journals database rather than creating the `related_bibstems.json` file by hand, so I will keep this in mind when I automate this.